.github/workflows/spark-ci.yml

```diff
@@ -83,7 +83,7 @@ jobs:
     strategy:
       matrix:
         jvm: [8, 11]
-        spark: ['3.0', '3.1', '3.2']
+        spark: ['3.0', '3.1', '3.3']
```

Comment:
Should add in 3.3 here and not replace 3.2

Reply:
I believe this is showing up in this [commit](https://github.com/apache/iceberg/pull/5094/commits/38eb20e7d1c7fcde22ec6ed79679b7a21ef12896) diff, In the final file diff including all the commits the 3.3 is added after 3.2 [complete diff](https://github.com/apache/iceberg/pull/5094/files#diff-cab7395ccb3f096361a9a1a6af739dad164d06383eae8742d26e11d7495c93d8). 

I replaced 3.2 to 3.3 in that [commit](https://github.com/apache/iceberg/pull/5094/commits/38eb20e7d1c7fcde22ec6ed79679b7a21ef12896)  as I was thinking this would be merge sequence https://github.com/apache/iceberg/pull/5094#issuecomment-1162859934 . Hence this showed up in this way.